apps/integration/src/app/integrations/modal-colorpicker/modal-colorpicker.module.ts

```diff
@@ -0,0 +1,13 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+
+import { ModalColorpickerRoutingModule } from './modal-colorpicker-routing.module';
+import { ModalColorpickerComponent } from './modal-colorpicker.component';
+
+@NgModule({
+  declarations: [ModalColorpickerComponent],
+  imports: [CommonModule, ModalColorpickerRoutingModule],
+})
+export class ModalColorpickerModule {
+  public static routes = ModalColorpickerRoutingModule.routes;
```

Comment:
☝️ Nearly all of the public properties and methods (on the new files in this PR) can use `protected` instead.